docs/IntegrationWithExistingApps.md

```diff
@@ -599,10 +605,10 @@ This is only really used in dev mode when reloading JavaScript from the developm
 
 ## Add native code
 
-You need to add some native code in order to start the React Native runtime and get it to render something. To do this, we're going to create an `Activity` that creates a `ReactRootView`, starts a React application inside it and sets it as the main content view.
+You need to add some native code in order to start the React Native runtime and get it to render something. To do this, we're going to create an `AppCompatActivity` that creates a `ReactRootView`, starts a React application inside it and sets it as the main content view.
```

Comment:
@jankalfus Yes, better to add this as a note since `AppCompatActivity` is not a necessity.